src/ModelTypes.h

```diff
@@ -100,49 +92,57 @@ struct msMaterial
 
 struct msPositionKey
 {
-	float fTime;
+	float fTime = 0;
 	RageVector3 Position;
+
+	msPositionKey() : fTime(0.0f) {} // Ensure fTime is initialized
 };
 
 struct msRotationKey
 {
-	float fTime;
+	float fTime = 0;
 	RageVector4 Rotation;
+
+	msRotationKey() : fTime(0.0f) {} // Ensure fTime is initialized
 };
 
 struct msBone
 {
-	int			nFlags;
-	RString			sName;
-	RString			sParentName;
-	RageVector3		Position;
-	RageVector3		Rotation;
+	int nFlags = 0;
```

Comment:
You can remove the `= 0` since we have it in the constructor.

Reply:
> You can remove the `= 0` since we have it in the constructor.

Done